# RAD
# https://github.com/diydrones/ardupilot/blob/master/libraries/DataFlash/DataFlash.h#L400
Header header

uint8 RSSI       # uint8_t rssi;
uint8 RemRSSI    # uint8_t remrssi;
uint8 TxBuf      # uint8_t txbuf;
uint8 Noise      # uint8_t noise;
uint8 RemNoise   # uint8_t remnoise;
uint16 RxErrors  # uint16_t rxerrors;
uint16 Fixed     # uint16_t fixed;
